graylog2-web-interface/src/components/common/ISODurationInput.jsx

```diff
@@ -14,6 +14,8 @@ const ISODurationInput = React.createClass({
     required: React.PropTypes.bool,
   },
 
+  mixins: [ISODurationUtils],
```

Comment:
Why are we using `ISODurationUtils` as a mixin? I think it makes less obvious where the methods are coming from, and in this case the mixin is not using any hooks or working with the component state.